src/components/UserInfo.js

```diff
@@ -9,18 +9,16 @@ export default class UserInfo {
     const userInfo = {
       name: this._nameElement.textContent,
       job: this._jobElement.textContent,
-      avatar: this._avatarElement.textContent,
     };
     return userInfo;
   }
 
-  setUserInfo(name, job, avatar) {
+  setUserInfo(name, job) {
     this._nameElement.textContent = name;
     this._jobElement.textContent = job;
-    this._avatarElement.textContent = avatar;
   }
 
-  setUserAvatar(link) {
-    this._avatarElement.src = link;
+  setUserAvatar(avatar) {
+    this._avatarElement.src = avatar;

```

Comment:
Every tag `img` should have an informative attribute `alt` (the **name** of the image is usually there) to show the `alternative` text if the image is not uploaded and it’s needed for **screen readers**


```js
  setUserAvatar(avatar) {
    this._avatarElement.src = avatar;
    this._avatarElement.alt = this._nameElement.textContent;  // we get the name of the user as `alt`
  }

```